docs/grafana.rst

```diff
@@ -0,0 +1,320 @@
+
+Requirements
+"""""""""""""""""""
+
+* Mac hardware must be a 2010 or newer model
+* macOS must be version 10.13 or newer
+* VirtualBox prior to version 4.3.30 must not be installed as it is not compatible with Docker Desktop.
+
+
+Installation
+"""""""""""""""""""
+
+There is an official `installation guide for Mac users <https://docs.docker.com/docker-for-windows/install>`_.
+
+Otherwise download and install `Docker CE for Mac (Stable) <https://download.docker.com/mac/stable/Docker.dmg>`_.
+
+
+Windows users
+^^^^^^^^^^^^^^^^^
+
+Requirements
+"""""""""""""""""""
+
+* Windows 10 64-bit (Pro, Enterprise, or Education (Build 15063 or later)) - these are pre-req for Docker Desktop.
+* Enable Hyper-V & Containers Windows features (usually done by docker installer).
+* BIOS-level hardware virtualization support must be enabled in the BIOS settings (here’s a `guide <#virtualization>`_ on how to do it)
+
+Installation
+"""""""""""""""""""
+
+There is an official `installation guide for Windows users <https://docs.docker.com/docker-for-windows/install>`_.
+
+Otherwise download and install `Docker CE for Windows (Stable) <https://download.docker.com/win/stable/Docker%20for%20Windows%20Installer.exe>`_.
+
+Virtualization
+""""""""""""""""""
+
+Windows users should also check if virtualization is enabled in firmware. To do so, check the Performance tab on the Task Manager:
+
+.. figure:: _static/figs/grafana-docker-windows-virtualization.png
+
+    check if virtualization is enabled
+
+If not, virtualization should be enabled in BIOS:
+
+* Reboot your computer
+*
+
+    Right when the computer is coming up from the black screen, press Delete, Esc, F1, F2, or F4. Each computer manufacturer uses
+    a different key but it may show a brief message at boot telling you which one to press. If you miss it the first time, reboot and try again.
+    It helps to tap the key about twice a second when the computer is coming up.
+
+* In the BIOS settings, find the configuration items related to the CPU. These can be in under the headings Processor, Chipset, or Northbridge.
+* Enable virtualization; the setting may be called VT-x, AMD-V, SVM, or Vanderpool. Enable Intel VT-d or AMD IOMMU if options are available.
+* Save your changes and reboot.
+
+Another `instruction with examples for different computer manufacturers <https://2nwiki.2n.cz/pages/viewpage.action?pageId=75202968>`_.
+
+Verify docker installation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Verify that docker is installed correctly by typing in terminal or in a command window::
+
+    docker run hello-world
+
+Output should contain::
+
+    Hello from Docker!
+    This message shows that your installation appears to be working correctly.
+
+
+Control Grafana using gradle commands
+----------------------------------------------
+
+Start Grafana
+^^^^^^^^^^^^^^^^^
+
+To run Grafana one should use gradle command grafanaStart without any parameters. The command will start Grafana and InfluxDB docker container,
```

Comment:
Is there a way to add inline code in rst? If so, `grafanaStart` should be wrapped

Reply:
`The command will start the Grafana...`